rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMClientExample.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive_grid_cloud_portal.rm.client;
+
+import java.security.KeyException;
+
+import javax.security.auth.login.LoginException;
+
+import org.ow2.proactive.resourcemanager.exception.RMException;
+import org.ow2.proactive_grid_cloud_portal.common.RMRestInterface;
+
+
+public class RMClientExample {
+
+    public static void main(String[] args) throws LoginException, KeyException, RMException {
+        System.out.println("Hello from example.");
+
+        //        HttpClientBuilder httpClientBuilder = HttpClientBuilder.create().useSystemProperties();
+        RMRestClient client = new RMRestClient("http://localhost:8080/rest/", null);//, new ApacheHttpClient4Engine(httpClientBuilder.build()));
```

Comment:
Explain why you keep the commented code or remove it please.

Reply:
This class is called *Example.
It is for us

Reply:
Every comments are for us. I don't understand your argument.

Reply:
There are 2 ways of creating the objects. One way commented out, and another is used.

Reply:
Can you add a comment explaining that please. I think it should be crystal clear for other developers ?